packages/components/src/unit-control/utils.ts

```diff
@@ -282,32 +288,31 @@ export function filterUnitsWithSettings(
  * @param  args.availableUnits Collection of unit value strings for filtering available units.
  * @param  args.defaultValues  Collection of default values for defined units. Example: { px: '350', em: '15' }.
  *
- * @return Filtered units based on settings.
+ * @return Filtered units of units, with their default values updated following the `defaultValues` argument's property.
```

Comment:
```suggestion
 * @return Filtered list of units, with their default values updated following the `defaultValues` argument's property.
```

...I think is what we mean?

Reply:
Haha yep! I'm not even sure what "units of units" may be